src/containers.js

```diff
@@ -15,6 +17,8 @@ const validComposeFile = async (contents) => {
   return valid;
 };
 
+const backupIsEnabled = () => process.env.CHT_BACKUP_COMPOSE_YML !== 'false';
```

Comment:
I would go for strict equal to `true` rather than strict difference to `false`

Reply:
That would change default behaviour to off, is that desired?

Reply:
I guess you're right. We want default behavior to be on.

Reply:
By the way is `true`/`false` preferable to `1`/`0`? (or `yes`/`no`, `on`/`off` but I'd prefer to avoid those as they tend to increase confusion)

Reply:
> By the way is true/false preferable to 1/0?

I would say yes.